src/openApi/openApiWrapper.ts

```diff
@@ -55,9 +57,21 @@ export default class OpenApiWrapper {
             user: this.userPrincipal,
             query: request.multiValueQueryStringParameters,
           });
+
+          if (config?.enableNewRelicTracking) {
+            const newrelic = await import('newrelic');
```

Comment:
Will this import run on every request? I'm not sure what optimization node.js does when importing the same module.

If there is no out-of-box optimization, you can make `newrelic` a member property and then first check `if(!this.newRelic) { this.newrelic = await import('newrelic') }

Reply:
Regardless of how it works, I like this idea!

Reply:
Aand done!